### PR TITLE
Create info.ini

### PR DESCRIPTION
**Pull request by witoldszpur**

### CODE REVIEW

**Comment by witoldszpur** [comment]
Add jquery.cropit

File: files/jquery.cropit/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Scott Cheng, Yufei Liu"
+github = "https://github.com/scottcheng/cropit"
+homepage = "http://scottcheng.github.io/cropit/"
+description = "Customizable crop and zoom."
+mainfile = "dist/jquery.cropit.js"
```
Comment (jimaek) [inline files/jquery.cropit/info.ini]: Just `jquery.cropit.js`

File: files/jquery.cropit/update.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "cropit",
+  "repo": "scottcheng/cropit",
+  "files": {
+    "include": ["jquery.cropit.js"]
```
Comment (jimaek) [inline files/jquery.cropit/update.json]: Here `dist/jquery.cropit.js`

**Comment by witoldszpur** [comment]
@jimaek THX, done.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**